src/app/app.component.html

```diff
@@ -113,8 +115,9 @@
                 [class.side-result-list-open]="listOpen">
                 <mat-tab-group [selectedIndex]="selectedListTabIndex" #tabsList>
                   <mat-tab *ngFor="let list of rightListContributors" [label]="$any(list).name">
-                    <div *ngIf="listOpen">
-                      <arlas-result-list #resultsidenav  style="height:100%" [rowItemList]="list.data" [fieldsList]="list.fieldsList"
+                    <div style="height: calc(100vh - 50px);
+                    overflow: hidden;">
```

Comment:
It shoud be more understandable in the future, if you add a class to this div instead of adding css in html

Reply:
you're right, I added it